column.go

```diff
@@ -35,6 +35,7 @@ func (l *BufferLen) Bind(h api.SQLHSTMT, idx int, ctype api.SQLSMALLINT, buf []b
 // Column provides access to row columns.
 type Column interface {
 	Name() string
+	DatabaseTypeName() string
```

Comment:
These changes require a new test in `mssql_test.go` that demonstrates / verifies new functionality. Otherwise it is impossible for me to judge if new coded works or not.

If you don't have MS SQL Server to test with, see Makefile with instructions on how to run MS SQL Server in Docker on Linux.